modules/raft/src/integrationTest/java/org/apache/ignite/internal/raft/ItLearnersTest.java

```diff
@@ -73,19 +77,38 @@
  */
 @ExtendWith(ConfigurationExtension.class)
 public class ItLearnersTest extends IgniteAbstractTest {
-    private static final String RAFT_GROUP_NAME = "test";
+    private static final ReplicationGroupId RAFT_GROUP_ID = new ReplicationGroupId() {
+        @Override
+        public String toString() {
+            return "test";
+        }
+    };
 
     private static final List<NetworkAddress> ADDRS = List.of(
             new NetworkAddress("localhost", 5000),
             new NetworkAddress("localhost", 5001),
             new NetworkAddress("localhost", 5002)
     );
 
+    private static final TestRaftMessagesFactory msgFactory = new TestRaftMessagesFactory();
```

Comment:
Constants should be named in upper case

Reply:
Technically this is not a constant, it's a static factory)

Reply:
You know what I mean)